From 638959581b0967db72823d936720c1b48a15149e Mon Sep 17 00:00:00 2001
From: Marko Lindqvist <cazfi74@gmail.com>
Date: Wed, 18 Oct 2023 08:25:13 +0300
Subject: [PATCH 29/29] Fix dbv_copy()

It was overwriting the pointer, not pointed memory.

See osdn #48869

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
---
 utility/bitvector.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utility/bitvector.c b/utility/bitvector.c
index 17a65411e6..d0895d5499 100644
--- a/utility/bitvector.c
+++ b/utility/bitvector.c
@@ -225,7 +225,7 @@ void dbv_copy(struct dbv *dest, const struct dbv *src)
     dbv_resize(dest, src->bits);
   }
 
-  memcpy(&dest->vec, src->vec, _BV_BYTES(src->bits));
+  memcpy(dest->vec, src->vec, _BV_BYTES(src->bits));
 }
 
 /***********************************************************************//**
-- 
2.42.0

